services/src/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperVirtualImpl.java

```diff
@@ -30,15 +35,29 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+/**
+ * Simulates the same functionality as {@link LockKeeperServiceImpl} but for virtual Mininet network.
+ */
 @Service
 @Profile("virtual")
 @Slf4j
-public class ASwitchVirtualImpl implements ASwitchService {
+public class LockKeeperVirtualImpl implements LockKeeperService {
+    private static final String FL_CONTAINER_NAME = "/floodlight";
```

Comment:
Can it be config option?

Reply:
Need to investigate how this is being configured during deployment. If it is not configurable there, then why bother. We do have couple more places with similar hardcoded values (e.g. speaker address when creating mininet topology). Will do this as a separate pr